web/src/components/Seo/Seo.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
```

Comment:
Could we remove this react import, please?

The redwood framework does some magic to add react without explicitly importing it. and why there's nothing wrong with importing it anyway, it makes it inconsistent with the rest of the react components.

Reply:
right, I will remove